### DISCUSSION TITLE
Welcome to Dune-II---The-Maker Discussions!

### DISCUSSION THREAD

**Discussion by stefanhendriks** [stefanhendriks/Dune-II---The-Maker#95] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MuadDib21** [comment]
Hi Stefan;
I wanted to ask you if you could tell me how to install your game. I've installed D2Tm file but don't know what to do next. I couldn't find anything on the internet and would really appreciate your help. Hope you're having a nice day ^^

**Reply by stefanhendriks** [reply]
Hi. Its fairly straightforward. Download the zip file. Unpack it somewhere. (ie suppose you have `c:\Games` you unpack there and you get `c:\Games\d2tm`). Then from that folder, open `d2tm.exe` or `windowed.bat`. And then enjoy! :)

**Comment by RobertFeyerharm** [comment]
Stefan,

Thank you for sharing your game! I've downloaded & unzipped to my PC. 

When I click on the d2tm.exe icon the game menu opens, but there is no mouse cursor to select items in the menu. Am I missing something?

Robert

**Reply by stefanhendriks** [reply]
Hi! Did you try these instructions? -> https://github.com/stefanhendriks/Dune-II---The-Maker/wiki/FAQ#game-does-not-start-windows-10

**Comment by rolanddenk** [comment]
Hi Stefan, firstly very impressive work you have done already with Dune II - The Maker, I really enjoy playing it, a great reminiscence of my childhood but with a lot of improvements :-)

I have some questions: 

- Is there a way to compile this on MacOS (x64 Sonoma)? 
- How can I exchange the sound files? Because I've created my own "Austrian Edition" sounds out of the original GERMAN.PAK Dune II sound files. They are working well with the original Dune II version and I'd love to enjoy them also in The Maker version.

Kind regards
Roland

**Reply by stefanhendriks** [reply]
Hi thanks for the compliments!

For the moment you can't compile it for Mac. I believe Allegro (the library used for gfx and sound and such) is not compilable for Mac. I intend to migrate to SDL ... at some point. Then it should be possible to compile for mac.

As for the sound files. You can run a tool called "grabber". I believe it is within the repo. It is a windows program you can run which you can use to manipulate .dat files. Just be sure you don't change the order of the files (also don't change the name or types).

**Comment by rolanddenk** [comment]
Hi Stefan ,thanks for your reply and information. It’s so great to see how much work you invest in that project. Looking forward to any future updates of the game.Best regards Roland Am 23.05.2024 um 20:23 schrieb Stefan Hendriks ***@***.***>:﻿
Hi thanks for the compliments!
For the moment you can't compile it for Mac. I believe Allegro (the library used for gfx and sound and such) is not compilable for Mac. I intend to migrate to SDL ... at some point. Then it should be possible to compile for mac.
As for the sound files. You can run a tool called "grabber". I believe it is within the repo. It is a windows program you can run which you can use to manipulate .dat files. Just be sure you don't change the order of the files (also don't change the name or types).
